static/src/users/UserCreate.vue

```diff
@@ -0,0 +1,116 @@
+            <label class="form-label">Prénom<span class="form-required"></span></label>
+            <input type="text" class="form-control" v-bind:class="{ 'state-invalid': errors.first_name }" v-model="formData.first_name">
+            <p class="text-muted pl-2" v-if="errors.first_name"><i class="fa fa-warning"></i> {{ errors.first_name.join(' / ')}}</p>
+          </div>
+          <div class="form-group">
+            <label class="form-label">Nom<span class="form-required"></span></label>
+            <input type="text" class="form-control" v-bind:class="{ 'state-invalid': errors.last_name }" v-model="formData.last_name">
+            <p class="text-muted pl-2" v-if="errors.last_name"><i class="fa fa-warning"></i> {{ errors.last_name.join(' / ')}}</p>
+          </div>
+          <div class="form-group">
+            <label class="form-label">Email<span class="form-required"></span></label>
+            <input type="email" class="form-control" v-bind:class="{ 'state-invalid': errors.email }" v-model="formData.email">
+            <p class="text-muted pl-2" v-if="errors.email"><i class="fa fa-warning"></i> {{ errors.email.join(' / ')}}</p>
+          </div>
+          <div class="form-group">
+            <label class="form-label">Organisme<span class="form-required"></span></label>
+            <input type="text" class="form-control" v-bind:class="{ 'state-invalid': errors.organization }" v-model="formData.organization">
+            <p class="text-muted pl-2" v-if="errors.organization"><i class="fa fa-warning"></i> {{ errors.organization.join(' / ')}}</p>
+          </div>
+        </fieldset>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal">Anuler</button>
+        <button type="submit" class="btn btn-primary">Ajouter</button>
+      </div>
+    </form>
+    </div>
+  </div>
+</div>
+</template>
+
+<script lang="ts">
+  import Vue from "vue";
+
+  import axios from 'axios'
+  import VueAxios from 'vue-axios'
+
+  import EventBus from '../events.js';
+
+  Vue.use(VueAxios, axios)
+
+  axios.defaults.xsrfCookieName = 'csrftoken'
+  axios.defaults.xsrfHeaderName = "X-CSRFTOKEN"
+
+  export default Vue.extend({
+    props: {
+      controlId: Number
+    },
+    data: () {
+      return {
+        'formData': {
+            'first_name': '',
+            'last_name': '',
+            'email': '',
+            'organization': '',
+            'controls': [],
+            'profile_type': ''
+        }
+        'postResult': [],
+        'errors': [],
+        'hasErrors': false
+      }
+    },
+    methods: {
+      hideModal() {
+        $('.modal-add-user').modal('hide');
+      },
+      addUser() {
+        this.formData.controls.push(this.controlId)
+        this.axios.post('/api/user/', this.formData)
+          .then(response => {
+            this.postResult = response.data
+            EventBus.$emit('user-added', this.postResult);
```

Comment:
Je regarde comment on pourrait éviter d'utiliser le EventBus. 🤔 (si tu veux te debarasser de ces PR on peut les merger, et ouvrir une PR separee pour ca. Apres tout le EventBus marche, donc on peut le merger sans casser develop.) 

En theorie ca devrait marcher si tu fais : 
- `$emit('user-added', this.postResult)` (sans EventBus ca devrait emitter sur le UserCreate lui-meme) 
-  dans le code du parent (User), tu ajoutes une ref sur le child : `<user-create ref="createUserChild">` 
- dans le code du parent (User), tu peux mettre le listener sur le child en utilisant la ref : `this.$refs.createUserChild.$on('user-added', blabla)`

Reply:
Sources :
https://vuejs.org/v2/guide/components-edge-cases.html#Accessing-Child-Component-Instances-amp-Child-Elements

Reply:
Il faudrait essayer et voir si ça marche. Je laisse le event bus pour l'instant.